### ISSUE TITLE
Verbosity and blanks should be published on the team page

### ISSUE THREAD

**Issue by krzyk** [zerocracy/farm#1184]
Because those metrics are gather for all users we should be show them on the team page (and they use their own xml files so this shouldn't slow down that page).
Those metrics might be later used for voting on jobs so it is better to have more eyes on them to catch potential errors in implementation.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by krzyk** [comment]
@g4s8 what do you think?

**Comment by krzyk** [comment]
@g4s8 Just waiving my hand here  :) If this doesn't make sense I'll close it (but I think it would be good for users to know and debug such things).

**Comment by g4s8** [comment]
@krzyk I think team page is already slightly complex: we have 7 columns here now. Also this page has some caching issues (loading user metrics for each user on demand), we are trying to fix it now, but each new field requires a lot of work to be rendered faster: change `people.xsd` in datum, change `People` class, add new stakeholder to publish each field to `people.xml`, therefore this task will be expensive (at least 3 tickets to resolve). So I would try to avoid new fields on this page for now, maybe later it will be actual.

**Comment by krzyk** [comment]
@g4s8 Yeah, it is slow right now.
How about if we add this information on users public profile page?

**Comment by g4s8** [comment]
@krzyk yes, it may be helpful, let's show these metrics on profile page. Please update the title.

**Comment by krzyk** [comment]
@g4s8 updated

**Comment by 0crat** [comment]
Job [#1184](https://github.com/zerocracy/farm/issues/1184) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)
